### PR TITLE
Validate Conf.json 

### PR DESCRIPTION
**Pull request by dorschw**
Fix values that would otherwise fail the [new conf.json validations](https://github.com/demisto/demisto-sdk/pull/4051)
Green build: https://github.com/demisto/content/actions/runs/8038145989/job/21953739890#step:7:507
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9228

### CODE REVIEW

File: Tests/conf.json [hunk 1]
```diff
@@ -5764,8 +5726,7 @@
         "get_original_email_-_ews-_test": "Issue 27571",
         "Trend Micro Deep Security - Test": "outsourced",
         "Microsoft Teams - Test": "Issue 38263",
-        "EWS Extension: Powershell Online V2 Test": "Issue 39008",
-        "O365 - EWS - Extension - Test": "Issue 39008",
```
Comment (sapirshuker) [inline Tests/conf.json:5768]: Where is the `O365 - EWS - Extension - Test` test?

File: Tests/conf.json [hunk 2]
```diff
@@ -6217,13 +6173,9 @@
         "Whois",
         "abuse.ch SSL Blacklist Feed",
         "ipinfo",
-        "ipinfo_v2",
```
Comment (sapirshuker) [inline Tests/conf.json:6220]: Is there a reason why you deleted it?

File: Tests/conf.json [hunk 3]
```diff
@@ -5419,10 +5389,6 @@
             "playbookID": "McAfee ePO v2 Test",
             "memory_threshold": 100
         },
-        {
-            "playbookID": "Dedup - Generic v3",
```
[the remark this answers is not in the document]
Comment (dorschw) [reply Tests/conf.json:5423]: deprecated

File: Tests/conf.json (hunk 1 above)
Comment (dorschw) [reply Tests/conf.json:5768]: duplicate key, edited, thx

**Review by yuvalbenshalom — COMMENTED** [review]
looks good.
see my question

File: Packs/CommonScripts/Scripts/GridFieldSetup/GridFieldSetup.yml [hunk 4]
```diff
@@ -51,4 +51,4 @@ tags:
 type: python
 fromversion: 6.5.0
 tests:
-- GridFieldSetup_test
```
[the remark this answers is not in the document]
Comment (dorschw) [reply Packs/CommonScripts/Scripts/GridFieldSetup/GridFieldSetup.yml:54]: No, that's the name of the unit tests file 🥲

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**